todoapp/app/src/test/java/com/example/android/architecture/blueprints/todoapp/statistics/StatisticsViewModelTest.java

```diff
@@ -78,18 +78,13 @@ public void getStatistics_withTasks_returnsCorrectData() {
         //Given a list of tasks in the repository
         when(mTasksRepository.getTasks()).thenReturn(Observable.just(mTasks));
         // And string resources
-        withText(R.string.statistics_active_tasks, ACTIVE_TASKS);
-        withText(R.string.statistics_completed_tasks, COMPLETED_TASKS);
+        withFormattedText(R.string.statistics_active_completed_tasks, ACTIVE_COMPLETED_TASKS);
```

Comment:
This seems to be unneeded. If you would remove this line, the test would work as well. 
You are testing if `ResourceProvider` was called, not it the value was used